Lib/types.py

```diff
@@ -143,6 +143,35 @@ def _calculate_meta(meta, bases):
                         "of the metaclasses of all its bases")
     return winner
 
+
+def get_original_bases(tp, /) -> tuple[type, ...] | None:
+    r"""
```

Comment:
Why is this a raw string?

```suggestion
    """
```

Reply:
Otherwise you get 
<img width="910" alt="image" src="https://user-images.githubusercontent.com/50501825/230425545-64efd120-e272-48bf-9cac-8a7249637c18.png">

Reply:
CPython docstrings don't need to be valid ReST, since our Sphinx docs are in separate files. Maybe just use single backticks? (Or just lose the backticks?)